pkg/ui/console/client.go

```diff
@@ -78,9 +78,18 @@ func (c Client) DownloadProviders(ctx context.Context) error {
 }
 
 func (c Client) Fetch(ctx context.Context, failOnError bool) error {
-	if err := c.DownloadProviders(ctx); err != nil {
-		return err
+	switch viper.GetBool("skip-schema-upgrade") {
+	case true:
```

Comment:
Weird style for a basic if-else. I wouldn't say the readability is improved by using a `switch` here.